eslint.config.js

```diff
@@ -0,0 +1,31 @@
+const typescriptParser = require('@typescript-eslint/parser')
+const typescriptPlugin = require('@typescript-eslint/eslint-plugin')
+const globals = require('globals')
+const eslint = require('@eslint/js')
+const tseslint = require('typescript-eslint')
+
+module.exports = [
+  eslint.configs.recommended,
+  ...tseslint.configs.recommended,
+  {
+    languageOptions: {
+      parser: typescriptParser,
+      globals: {
+        ...globals.node,
+        ...globals.es2015
```

Comment:
Do we need this `globals.es2015` option?

Reply:
We had `es6: true` in the previous config file which is the same as `globals.es2015`